diff --git a/PKGBUILD b/PKGBUILD
index 90cd26a..ba6fea6 100755
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Antonio Arias Orzanco <antonio dot arias99999 at gmail dot com>
 
 pkgname=scrivano
-pkgver=0.21.5
+pkgver=0.23.5
 pkgrel=1
 pkgdesc="Notetaking application aimed at those who prefer handwriting over typing."
 arch=('x86_64')
@@ -11,7 +11,7 @@ license=('CC-BY-NC-ND-4.0')
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/scrivanolabs/ScrivanoForLinux/releases/download/${pkgver}/Scrivano_${pkgver}.tar.gz"
          scrivano.svg::https://raw.githubusercontent.com/scrivanolabs/scrivanolabs.github.io/master/icon.svg
          scrivano.desktop)
-md5sums=('909bb4bc8ba5bfdfed135c3da5f303aa'
+md5sums=('25391ff12c3645ea4e6a066a71617a99'
          '4a39e18509d8cc877d4934c39ec14d5d'
          '4ca6be9398d9c8339d3d51a5e4bbf152')
 
